Minor typo fix in zap_labels function in labelled.R

markriseley opened the pull request:
zap_labels vignette doesn't work as intended currently. I think it's just a missing "!" :zap_labels is meant to act on class "labelled" not return it unmodified, as I understand it.

hadley commented:
Doh! Would you mind also adding a unit test?

markriseley commented:
2 tests added now. Does that work for you?

hadley commented:
Perfect - thanks!  Would you mind also adding a bullet point to NEWS, thanking your github user name?

markriseley commented:
Sure, I added this under .9000 in news.MD:


   - fixed bug in zap_labels() which was leaving labelled vectors unchanged
   instead of leaving unlabelled vectors unchanged: thanks to markriseley.


On 28 April 2015 at 13:22, Hadley Wickham <notifications@github.com> wrote:

> Perfect - thanks! Would you mind also adding a bullet point to NEWS,
> thanking your github user name?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hadley/haven/pull/67#issuecomment-97042584>.
>

hadley commented:
Thanks!
